src/components/ReportActionItem/MoneyRequestPreview.tsx

```diff
@@ -317,31 +301,31 @@ function MoneyRequestPreview(props) {
                                         </View>
                                     )}
                                 </View>
-                                {props.isBillSplit && (
+                                {isBillSplit && (
                                     <View style={styles.moneyRequestPreviewBoxAvatar}>
                                         <MultipleAvatars
                                             icons={sortedParticipantAvatars}
                                             shouldStackHorizontally
                                             size="small"
-                                            isHovered={props.isHovered}
+                                            isHovered={isHovered}
                                             shouldUseCardBackground
                                         />
                                     </View>
                                 )}
                             </View>
                             <View style={[styles.flexRow, styles.mt1]}>
                                 <View style={[styles.flex1]}>
-                                    {!isCurrentUserManager && props.shouldShowPendingConversionMessage && (
+                                    {!isCurrentUserManager && shouldShowPendingConversionMessage && (
                                         <Text style={[styles.textLabel, styles.colorMuted]}>{translate('iou.pendingConversionMessage')}</Text>
                                     )}
                                     {shouldShowDescription && <RenderHTML html={parser.replace(merchantOrDescription)} />}
                                     {shouldShowMerchant && <Text style={[styles.textLabelSupporting, styles.textNormal]}>{merchantOrDescription}</Text>}
                                 </View>
-                                {props.isBillSplit && !_.isEmpty(participantAccountIDs) && requestAmount > 0 && (
+                                {isBillSplit && participantAccountIDs.length > 0 && requestAmount && requestAmount > 0 && (
```

Comment:
@VickyStash Native app crashes on this line. Can you please raise quick PR fixing that?
```suggestion
                                {isBillSplit && participantAccountIDs.length > 0 && !!requestAmount && requestAmount > 0 && (
```
<img width="323" alt="Screenshot 2024-02-06 at 9 15 57 PM" src="https://github.com/Expensify/App/assets/108292595/e3b4e65a-3895-41e4-8861-0a0fbf0863a1">

Reply:
@situchan I'm taking a look

Reply:
@situchan could you please provide me steps for reproduction so I can be sure the issue is fixed?

Reply:
I tested that solution myself. Works well.
I was just testing https://github.com/Expensify/App/pull/35876. Please try to follow QA step on that PR.

Reply:
So that happens when `requestAmount` = 0
Try this:
A request $10 from B
B request $10 (same amount) from A

Reply:
@situchan I got what you mean and it makes sense to me, but I'm still not able to trigger this case on the main, could you please provide me reproduction video 🙏